### PR TITLE
Add GameSpanish class

### PR DESCRIPTION
**Pull request by leep3**
GameSpanish class is a child class of Game class.

I checked that if the user clicks the Spanish button then change into Spanish deck.

### CODE REVIEW

File: src/main/java/models/GameSpanish.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            }
+        }
+
+        //all columns exhausted, no valid matches found
+        return false;
+    }
+    public void check_possible_Move()                   //checks all possible moves. Returns true if there is still possible moves. Returns false if there is no more moves.
+    {
+
+        if (!columnHasCards(0))   //don't do anything if there is still a possible move. (move to empty column or remove from column)
```
Comment (liaoto) [inline src/main/java/models/GameSpanish.java]: Line 60 to 100 some spacing issues. Could condense it.

File: src/main/java/models/GameSpanish.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
+package models;
+
+import java.util.ArrayList;
+
+/**
+ * Assignment 1: Each of the blank methods below require implementation to get AcesUp to build/run
+ */
+public class GameSpanish extends Game{
```
Comment (milfordn) [inline src/main/java/models/GameSpanish.java]: In extending game, you don't need to copy all of the methods and fields; those are copied implicitly from the base class game. Anything that's the same as the base game class can and should be deleted.

**Comment by milfordn** [comment]
Looks good.

File: src/main/java/models/GameSpanish.java (hunk 2 above)
Comment (leep3) [reply src/main/java/models/GameSpanish.java]: I changed it!

**Pull request merged**